spec/datadog/tracing/span_operation_spec.rb

```diff
@@ -999,6 +999,107 @@
       end
     end
   end
+
+  describe '#record_exception' do
+    let(:error) { StandardError.new('test error').tap { |e| e.set_backtrace(['this is a backtrace']) } }
+
+    it 'creates a span event' do
+      begin
+        raise error
+      rescue => e
+        span_op.record_exception(e)
+        span_op.record_exception(e)
+      end
```

Comment:
To be honest this doesn't make sense. Since `record_exception` is not providing exception handling, we should just create exception, stuff it with what we want and give it to the method. Raising exception here do nothing